### PR TITLE
feat(plasma-tokens-utils): Update formatting tokens name from camelCase to kebab-case 

### PR DESCRIPTION
**Pull request by neretin-trike**
### Tokens

Изменён нейминг старых токенов цветов и типографики с нотации `camelCase` на `kebab-case`.

Было:
Screenshot 2024-01-16 at 19 09 54
Стало:
Screenshot 2024-01-16 at 19 09 17

### What/why changed

Библиотека styled-component в 5 версии автоматически переименовывала токены в `kebab-case` нотацию. Но новая 6 версия, а также другая css-in-js библиотека `Linaria` этого не делает. Поэтому для их поддержки это было сделано на нашей стороне

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.971.7542106012.ts'>borderRadius_default_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.971.7542106012.ts'>borderRadius_plasma_b2c_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.971.7542106012.ts'>borderRadius_plasma_web_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.971.7542106012.ts'>borderRadius_sberHealth_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.971.7542106012.ts'>borderRadius_sbermarket_business_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.971.7542106012.ts'>borderRadius_sbermarket_metro_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.971.7542106012.ts'>borderRadius_sbermarket_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.971.7542106012.ts'>borderRadius_sbermarket_selgros_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.971.7542106012.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.971.7542106012.ts'>borderRadius_sberonline_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.971.7542106012.ts'>borderRadius_sberprime_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.971.7542106012.ts'>borderRadius_stylesSalute_react-native--canary.971.7542106012.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.971.7542106012.swift'>color_default_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.971.7542106012.kt'>color_default_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.971.7542106012.ts'>color_default_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.971.7542106012.xml'>color_default_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.971.7542106012.swift'>color_plasma_b2c_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.971.7542106012.kt'>color_plasma_b2c_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.971.7542106012.ts'>color_plasma_b2c_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.971.7542106012.xml'>color_plasma_b2c_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.971.7542106012.swift'>color_plasma_web_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.971.7542106012.kt'>color_plasma_web_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.971.7542106012.ts'>color_plasma_web_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.971.7542106012.xml'>color_plasma_web_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.971.7542106012.swift'>color_sberHealth_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.971.7542106012.kt'>color_sberHealth_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.971.7542106012.ts'>color_sberHealth_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.971.7542106012.xml'>color_sberHealth_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.971.7542106012.swift'>color_sbermarket_business_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.971.7542106012.kt'>color_sbermarket_business_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.971.7542106012.ts'>color_sbermarket_business_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.971.7542106012.xml'>color_sbermarket_business_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.971.7542106012.swift'>color_sbermarket_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.971.7542106012.kt'>color_sbermarket_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.971.7542106012.swift'>color_sbermarket_metro_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.971.7542106012.kt'>color_sbermarket_metro_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.971.7542106012.ts'>color_sbermarket_metro_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.971.7542106012.xml'>color_sbermarket_metro_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.971.7542106012.ts'>color_sbermarket_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.971.7542106012.swift'>color_sbermarket_selgros_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.971.7542106012.kt'>color_sbermarket_selgros_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.971.7542106012.ts'>color_sbermarket_selgros_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.971.7542106012.xml'>color_sbermarket_selgros_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.971.7542106012.swift'>color_sbermarket_wlbusiness_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.971.7542106012.kt'>color_sbermarket_wlbusiness_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.971.7542106012.ts'>color_sbermarket_wlbusiness_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.971.7542106012.xml'>color_sbermarket_wlbusiness_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.971.7542106012.xml'>color_sbermarket_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.971.7542106012.swift'>color_sberonline_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.971.7542106012.kt'>color_sberonline_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.971.7542106012.ts'>color_sberonline_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.971.7542106012.xml'>color_sberonline_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.971.7542106012.swift'>color_sberprime_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.971.7542106012.kt'>color_sberprime_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.971.7542106012.ts'>color_sberprime_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.971.7542106012.xml'>color_sberprime_xml--canary.971.7542106012.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.971.7542106012.swift'>color_stylesSalute_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.971.7542106012.kt'>color_stylesSalute_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.971.7542106012.ts'>color_stylesSalute_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.971.7542106012.xml'>color_stylesSalute_xml--canary.971.7542106012.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.971.7542106012.ts'>shadow_default_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.971.7542106012.ts'>shadow_plasma_b2c_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.971.7542106012.ts'>shadow_plasma_web_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.971.7542106012.ts'>shadow_sberHealth_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.971.7542106012.ts'>shadow_sbermarket_business_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.971.7542106012.ts'>shadow_sbermarket_metro_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.971.7542106012.ts'>shadow_sbermarket_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.971.7542106012.ts'>shadow_sbermarket_selgros_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.971.7542106012.ts'>shadow_sbermarket_wlbusiness_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.971.7542106012.ts'>shadow_sberonline_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.971.7542106012.ts'>shadow_sberprime_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.971.7542106012.ts'>shadow_stylesSalute_react-native--canary.971.7542106012.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.971.7542106012.ts'>spacing_default_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.971.7542106012.ts'>spacing_plasma_b2c_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.971.7542106012.ts'>spacing_plasma_web_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.971.7542106012.ts'>spacing_sberHealth_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.971.7542106012.ts'>spacing_sbermarket_business_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.971.7542106012.ts'>spacing_sbermarket_metro_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.971.7542106012.ts'>spacing_sbermarket_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.971.7542106012.ts'>spacing_sbermarket_selgros_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.971.7542106012.ts'>spacing_sbermarket_wlbusiness_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.971.7542106012.ts'>spacing_sberonline_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.971.7542106012.ts'>spacing_sberprime_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.971.7542106012.ts'>spacing_stylesSalute_react-native--canary.971.7542106012.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.971.7542106012.swift'>typo_mage_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.971.7542106012.kt'>typo_mage_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.971.7542106012.ts'>typo_mage_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.971.7542106012.swift'>typo_plasma_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.971.7542106012.kt'>typo_plasma_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.971.7542106012.ts'>typo_plasma_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.971.7542106012.swift'>typo_ruler_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.971.7542106012.kt'>typo_ruler_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.971.7542106012.ts'>typo_ruler_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.971.7542106012.swift'>typo_sage_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.971.7542106012.kt'>typo_sage_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.971.7542106012.ts'>typo_sage_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.971.7542106012.swift'>typo_sbermarket_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.971.7542106012.kt'>typo_sbermarket_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.971.7542106012.ts'>typo_sbermarket_react-native--canary.971.7542106012.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.971.7542106012.swift'>typo_soulmate_ios-swift--canary.971.7542106012.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.971.7542106012.kt'>typo_soulmate_kotlin--canary.971.7542106012.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.971.7542106012.ts'>typo_soulmate_react-native--canary.971.7542106012.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-asdk@0.26.0-canary.971.7542106012.0
  npm install @salutejs/plasma-b2c@1.268.0-canary.971.7542106012.0
  npm install @salutejs/plasma-core@1.144.0-canary.971.7542106012.0
  npm install @salutejs/plasma-hope@1.254.0-canary.971.7542106012.0
  npm install @salutejs/plasma-icons@1.177.0-canary.971.7542106012.0
  npm install @salutejs/plasma-new-hope@0.32.0-canary.971.7542106012.0
  npm install @salutejs/plasma-temple@1.196.0-canary.971.7542106012.0
  npm install @salutejs/plasma-tokens-b2b@1.33.0-canary.971.7542106012.0
  npm install @salutejs/plasma-tokens-b2c@0.43.0-canary.971.7542106012.0
  npm install @salutejs/plasma-tokens-web@1.48.0-canary.971.7542106012.0
  npm install @salutejs/plasma-tokens@1.68.0-canary.971.7542106012.0
  npm install @salutejs/plasma-ui@1.228.0-canary.971.7542106012.0
  npm install @salutejs/plasma-web@1.268.0-canary.971.7542106012.0
  npm install @salutejs/plasma-cy-utils@0.76.0-canary.971.7542106012.0
  npm install @salutejs/plasma-sb-utils@0.142.0-canary.971.7542106012.0
  npm install @salutejs/plasma-tokens-utils@0.37.0-canary.971.7542106012.0
  # or 
  yarn add @salutejs/plasma-asdk@0.26.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-b2c@1.268.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-core@1.144.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-hope@1.254.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-icons@1.177.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-new-hope@0.32.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-temple@1.196.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-tokens-b2b@1.33.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-tokens-b2c@0.43.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-tokens-web@1.48.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-tokens@1.68.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-ui@1.228.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-web@1.268.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-cy-utils@0.76.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-sb-utils@0.142.0-canary.971.7542106012.0
  yarn add @salutejs/plasma-tokens-utils@0.37.0-canary.971.7542106012.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!

  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-971/

File: utils/plasma-tokens-utils/src/utils/objectToCSSVariables.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const objectToCSSVariables = (
 
             vars = {
                 ...vars,
-                ...objectToCSSVariables(value, name, withPrefixDesign),
+                ...objectToCSSVariables(value, name, withPrefixDesign, withKebabCase),
```
Comment (Yakutoc) [inline utils/plasma-tokens-utils/src/utils/objectToCSSVariables.ts:48]: [Minor]

Имхо, может нам сигнатуру функции на объектную перевести? Что бы было видно какие аргументы за что отвечают

Comment (neretin-trike) [reply utils/plasma-tokens-utils/src/utils/objectToCSSVariables.ts:48]: сильно больно получится, ибо много мест где это используется. В идеале от этого всего в будущем надо будет уйти, когда откажемся от пакетов с токенами

File: packages/plasma-tokens/src/typo/sberPortal.ts [hunk 2]
```diff
@@ -2,114 +2,114 @@
...
+        '--plasma-typo-headline1-font-size': '2rem',
+        '--plasma-typo-headline1-letter-spacing': '-0.0110em',
+        '--plasma-typo-headline1-line-height': '2.25rem',
+        '--plasma-typo-headline2-font-family': '"SB Sans Text","SBSansText","Helvetica","Arial",sans-serif',
+        '--plasma-typo-headline2-font-weight': '600',
+        '--plasma-typo-headline2-font-style': 'normal',
+        '--plasma-typo-headline2-font-size': '1.5rem',
+        '--plasma-typo-headline2-letter-spacing': '-0.0220em',
+        '--plasma-typo-headline2-line-height': '1.75rem',
+        '--plasma-typo-headline3-font-family': '"SB Sans Text","SBSansText","Helvetica","Arial",sans-serif',
```
[the remark this answers is not in the document]
Comment (neretin-trike) [reply packages/plasma-tokens/src/typo/sberPortal.ts:35]: в описании как раз есть это :) Линария и другие библиотеки (даже тот же styled-component 6) внутри себя не превращают токены в эту нотацию. А это в свою очередь будет ломаться, т.к. у нас везде используется kebab-case

**Review by Yakutoc — APPROVED** [review]

**Review by kayman233 — APPROVED** [review]

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.27.0-dev.0, @salutejs/plasma-b2c@1.269.0-dev.0, @salutejs/plasma-core@1.144.0-dev.0, @salutejs/plasma-cy-utils@0.76.0-dev.0, @salutejs/plasma-hope@1.255.0-dev.0, @salutejs/plasma-icons@1.177.0-dev.0, @salutejs/plasma-new-hope@0.33.0-dev.0, @salutejs/plasma-sb-utils@0.142.0-dev.0, @salutejs/plasma-temple-docs@0.207.0-dev.0, @salutejs/plasma-temple@1.196.0-dev.0, @salutejs/plasma-tokens-b2b@1.33.0-dev.0, @salutejs/plasma-tokens-b2c@0.43.0-dev.0, @salutejs/plasma-tokens-native@1.19.0-dev.0, @salutejs/plasma-tokens-utils@0.37.0-dev.0, @salutejs/plasma-tokens-web@1.48.0-dev.0, @salutejs/plasma-tokens@1.68.0-dev.0, @salutejs/plasma-ui-docs@0.256.0-dev.0, @salutejs/plasma-ui@1.228.0-dev.0, @salutejs/plasma-web-docs@0.223.0-dev.0, @salutejs/plasma-web@1.269.0-dev.0, @salutejs/plasma-website@0.231.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.27.0-dev.0, @salutejs/plasma-b2c@1.269.0-dev.0, @salutejs/plasma-core@1.144.0-dev.0, @salutejs/plasma-cy-utils@0.76.0-dev.0, @salutejs/plasma-hope@1.255.0-dev.0, @salutejs/plasma-icons@1.177.0-dev.0, @salutejs/plasma-new-hope@0.33.0-dev.0, @salutejs/plasma-sb-utils@0.142.0-dev.0, @salutejs/plasma-temple-docs@0.207.0-dev.0, @salutejs/plasma-temple@1.196.0-dev.0, @salutejs/plasma-tokens-b2b@1.33.0-dev.0, @salutejs/plasma-tokens-b2c@0.43.0-dev.0, @salutejs/plasma-tokens-native@1.19.0-dev.0, @salutejs/plasma-tokens-utils@0.37.0-dev.0, @salutejs/plasma-tokens-web@1.48.0-dev.0, @salutejs/plasma-tokens@1.68.0-dev.0, @salutejs/plasma-ui-docs@0.256.0-dev.0, @salutejs/plasma-ui@1.228.0-dev.0, @salutejs/plasma-web-docs@0.223.0-dev.0, @salutejs/plasma-web@1.269.0-dev.0, @salutejs/plasma-website@0.231.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.27.0-dev.0, @salutejs/plasma-b2c@1.269.0-dev.0, @salutejs/plasma-core@1.144.0-dev.0, @salutejs/plasma-cy-utils@0.76.0-dev.0, @salutejs/plasma-hope@1.255.0-dev.0, @salutejs/plasma-icons@1.177.0-dev.0, @salutejs/plasma-new-hope@0.33.0-dev.0, @salutejs/plasma-sb-utils@0.142.0-dev.0, @salutejs/plasma-temple-docs@0.207.0-dev.0, @salutejs/plasma-temple@1.196.0-dev.0, @salutejs/plasma-tokens-b2b@1.33.0-dev.0, @salutejs/plasma-tokens-b2c@0.43.0-dev.0, @salutejs/plasma-tokens-native@1.19.0-dev.0, @salutejs/plasma-tokens-utils@0.37.0-dev.0, @salutejs/plasma-tokens-web@1.48.0-dev.0, @salutejs/plasma-tokens@1.68.0-dev.0, @salutejs/plasma-ui-docs@0.256.0-dev.0, @salutejs/plasma-ui@1.228.0-dev.0, @salutejs/plasma-web-docs@0.223.0-dev.0, @salutejs/plasma-web@1.269.0-dev.0, @salutejs/plasma-website@0.231.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.29.0, @salutejs/plasma-b2c@1.271.0, @salutejs/plasma-core@1.144.0, @salutejs/plasma-cy-utils@0.76.0, @salutejs/plasma-hope@1.255.0, @salutejs/plasma-icons@1.177.0, @salutejs/plasma-new-hope@0.35.0, @salutejs/plasma-sb-utils@0.142.0, @salutejs/plasma-temple-docs@0.209.0, @salutejs/plasma-temple@1.196.0, @salutejs/plasma-tokens-b2b@1.33.0, @salutejs/plasma-tokens-b2c@0.43.0, @salutejs/plasma-tokens-native@1.19.0, @salutejs/plasma-tokens-utils@0.37.0, @salutejs/plasma-tokens-web@1.48.0, @salutejs/plasma-tokens@1.68.0, @salutejs/plasma-ui-docs@0.258.0, @salutejs/plasma-ui@1.228.0, @salutejs/plasma-web-docs@0.225.0, @salutejs/plasma-web@1.271.0, @salutejs/plasma-website@0.233.0 🚀

**Pull request merged**